config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  patch "/guests/:id", to: "guests#update"
```

Comment:
This PATCH route should be a POST, and the request should go to a joins controller since we are doing work on the joins table.

`post "/guests/:id/rooms`, to: "guest_rooms#create"`